### PR TITLE
Remove IP transaction check

### PR DESCRIPTION
**Pull request by ghost**
> The code checking for 'online' IP transactions can be removed.

~ laanwj https://github.com/bitcoin/bitcoin/issues/8530#issuecomment-240370094

I attempted to remove it and when I tested it, it compiled succesfully.

### CODE REVIEW

File: src/qt/transactiondesc.cpp [hunk 1]
```diff
@@ -180,22 +161,21 @@ QString TransactionDesc::toHTML(CWallet *wallet, CWalletTx &wtx, TransactionReco
                 if ((toSelf == ISMINE_SPENDABLE) && (fAllFromMe == ISMINE_SPENDABLE))
                     continue;
 
-                if (!wtx.mapValue.count("to") || wtx.mapValue["to"].empty())
+
+                // Offline transaction
```
Comment (jonasschnelli) [inline src/qt/transactiondesc.cpp]: Can we remove the term "offline transaction"? I think it's confusion.

Comment (mcccs) [inline src/qt/transactiondesc.cpp]: Sure. He confused me too.

**Comment by MarcoFalke** [comment]
Please get rid of the merge commit according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

You can try

```
git reset 67df90a~
git commit src/qt

**Comment by MarcoFalke** [comment]
Concept ACK!

**Comment by laanwj** [comment]
concept ACK, looks good to me, although I have not verified that this doesn't affect payment request functionality (does anything else set transaction metadata 'from' and 'to'?)

**Comment by luke-jr** [comment]
How does this affect a wallet that has received IP payments in the past?

**Comment by laanwj** [comment]
Yes, that should be tested, if anyone still has such a wallet around. Crashing would be unacceptable, showing it differently would be.

**Comment by MarcoFalke** [comment]
Please also get rid of the other `tr("From")`.

**Comment by laanwj** [comment]
BTW: do we do anything special in the transaction overview for these kind of transactions? if so, that code should go at the same time

**Comment by mcccs** [comment]
```bash
git reset 67df90a~
git commit src/qt
```

I tried this but they conflicted.

**Comment by MarcoFalke** [comment]
@mcccs Sry, I forgot to mention you also need to rebase on master after that. You can join the developer chat and we are happy to help, btw.

```
git rebase f4e777819c54fe0e386de517a201053698556236

**Comment by mcccs** [comment]
Why did it commit "Merge remote-tracking branch 'origin/Ip-check' into Ip-check"?
I didn't do that.

**Pull request closed without merging**